### ISSUE TITLE
Red

### ISSUE THREAD

**Issue by clefebvre** [linuxmint/mint-y-colors#9]
**Old**

![image](https://user-images.githubusercontent.com/1138515/77822593-2a26cc80-70ec-11ea-8186-211b94e43744.png)

**New**

![image](https://user-images.githubusercontent.com/1138515/77825681-c1971a00-7102-11ea-9f39-63008f87d3fa.png)

**Comment by clefebvre** [comment]
<>

**Comment by kelebek333** [comment]
New Mint-Y-Red is too warmed than old version. I think red color temperature can be lowered a bit.

**Comment by ghost** [comment]
The old one is just perfect.

**Comment by zboraon** [comment]
> New Mint-Y-Red is too warmed than old version. I think red color temperature can be lowered a bit.

I agree with kelebek333. The older one was not OK, but this one is too much eye tiring.

**Comment by kongreddgrass** [comment]
I'm agree with the others. The old red is not fine and the new is too strong.
What's about #8e0e1e?

**Comment by Nebelkind** [comment]
like the new one for folders and in combination with dark themes but it is slightly overwhelming in menues. Definitely an improvement to the old one though

**Comment by bluz71** [comment]
I prefer *old*.

**Comment by SebastJava** [comment]
![red](https://user-images.githubusercontent.com/31940713/88553019-cc9b8180-cff2-11ea-829a-55de2451a9a7.png)
This is not a distracting background color, this is the focus color.
So my focus color is... colored. And contrasted.
